tap_github/tests/__init__.py

```diff
@@ -11,9 +11,13 @@
     "api_calls_tests_cache",
     backend="sqlite",
     # make sure that API keys don't end up being cached
-    ignored_parameters=["Authorization"],
+    # Also ignore user-agent so that various versions of request
+    # can share the cache
+    ignored_parameters=["Authorization", "user-agent"],
```

Comment:
Nice find! Any other parameters we should ignore?

Reply:
I hope not. I looked at the data passed to the cache_key function, and this seemed to be the only value that might be annoying.